resources/META-INF/plugin.xml

```diff
@@ -20,6 +20,7 @@
   <!--Add gradle as a dep because of Pants runner requires it.-->
   <depends>com.intellij.gradle</depends>
 
+  <depends optional="true" config-file="pants-junit.xml">JUnit</depends>
```

Comment:
This should be required. so `optional=false` or not specifying it at all.

Reply:
I chose conditional dependency, because otherwise it was failing when you were using 2019.2 for development. 

Now it's unconditional.

Reply:
Thanks! The plugin upgrade is normally backwards incompatible, so not-optional should be fine.